xray/plot/plot.py

```diff
@@ -230,6 +230,102 @@ def hist(darray, ax=None, **kwargs):
     return primitive
 
 
+def _plotter_1d(darray, plot_method='plot', *args, **kwargs):
+    """
+    Bar plot of 1 dimensional DataArray index against values
+
+    Wraps matplotlib.pyplot.plot
+
+    Parameters
+    ----------
+    darray : DataArray
+        Must be 1 dimensional
+    ax : matplotlib axes, optional
+        If not passed, uses the current axis
+    *args, **kwargs : optional
+        Additional arguments to matplotlib.pyplot.plot
+
+    """
+    import matplotlib.pyplot as plt
+
+    ndims = len(darray.dims)
+    if ndims != 1:
+        raise ValueError('Line plots are for 1 dimensional DataArrays. '
+                         'Passed DataArray has {ndims} '
+                         'dimensions'.format(ndims=ndims))
+
+    # Ensures consistency with .plot method
+    ax = kwargs.pop('ax', plt.gca())
+
+    xlabel, x = list(darray.indexes.items())[0]
+
+    _ensure_plottable([x])
+
+    try:
+        plotter = getattr(ax, plot_method)
+    except AttributeError:
+        msg = 'Axes does not have plotting method {}'.format(plot_method)
+        raise AttributeError(msg)
+
+    primitive = plotter(x, darray, *args, **kwargs)
+
+    ax.set_xlabel(xlabel)
+    ax.set_title(darray._title_for_slice())
+
+    if darray.name is not None:
+        ax.set_ylabel(darray.name)
+
+    # Rotate dates on xlabels
+    if np.issubdtype(x.dtype, np.datetime64):
+        plt.gcf().autofmt_xdate()
+
+    return primitive
+
+
+def _plotter_data(darray, plot_method='hist', **kwargs):
+    """
+    Bar plot of 1 dimensional DataArray index against values
+
+    Wraps matplotlib.pyplot.plot
+
+    Parameters
+    ----------
+    darray : DataArray
+        Must be 1 dimensional
+    ax : matplotlib axes, optional
+        If not passed, uses the current axis
+    *args, **kwargs : optional
+        Additional arguments to matplotlib.pyplot.plot
+
+    """
+    import matplotlib.pyplot as plt
+
+    ndims = len(darray.dims)
```

Comment:
Could factor out common logic between `_plotter_1d` and this through a helper function?